input/ipc-win.c

```diff
@@ -464,6 +464,25 @@ struct mp_ipc_ctx *mp_init_ipc(struct mp_client_api *client_api,
         .client_api = client_api,
     };
 
+    if (opts->ipc_client && opts->ipc_client[0]) {
```

Comment:
I know it is copied from ipc-unix, but maybe we could reduce complexity, with helpers like bstr_startswith and bstrtoll?

Reply:
> How? The string needs to be wrapped and there is stilI no steps being saved with this.

See the attached code. I want to encourage people to use bstr more.

``` c++
    bstr str = bstr0(opts->ipc_client);
    if (bstr_eatstart0(&str, "fd://")) {
        int fd = -1;
        long long ll = bstrtoll(str, &str, 10);
        if (!str.len && ll <= INT_MAX)
            fd = ll;
        if (fd < 0) {
            MP_ERR(arg, "Invalid IPC client argument: '%s'\n", opts->ipc_client);
        } else {
            HANDLE h = (HANDLE)_get_osfhandle(fd);
            if (h && h != INVALID_HANDLE_VALUE && (intptr_t)h != -2) {
                ipc_start_client_json(arg, -1, h);
            } else {
                MP_ERR(arg, "Invalid IPC client fd: '%d'\n", fd);
            }
        }
    }
```